changelog/unreleased/breaking-changes/2096--remove-vast-get.md

```diff
@@ -0,0 +1 @@
+We removed the experimental `vast get` command.
```

Comment:
"Why?" would be the first question users ask themselves. I doesn't hurt to provide a short explanation.

Reply:
I realized that we don't actually need this change here, but I will open a dedicated PR soon.